plugins/package-managers/spdx/src/funTest/assets/projects/synthetic/dependson-packages/project-xyz.spdx.yml

```diff
@@ -0,0 +1,92 @@
+SPDXID: "SPDXRef-DOCUMENT"
```

Comment:
Nit: Let's use "DEPENDS_ON" instead of "dependson" in the path to resemble the enum name.

Reply:
Commit message title nit: Consider putting "DEPENDS_ON" into Markdown-style backticks to format it as a keyword.